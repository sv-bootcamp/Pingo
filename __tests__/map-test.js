import 'react-native';
import React from 'react';
import Map from '../shared/components/map';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Map 
      items={[]}
      getMapItems={()=>{}}
      selectedItem={{}}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
