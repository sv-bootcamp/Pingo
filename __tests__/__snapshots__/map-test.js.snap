exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "flex-end",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0
    }
  }>
  <View
    style={
      Array [
        Object {
          "alignSelf": "flex-start",
          "borderColor": "red",
          "borderWidth": 1
        },
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0
        }
      ]
    } />
</View>
`;
